website/docs/components/tag/partials/code/component-api.md

```diff
@@ -24,6 +24,9 @@
   <C.Property @name="isRouteExternal" @type="boolean" @default="false">
     Controls if the “LinkTo” is external to the Ember engine, in which case it will use a `<LinkToExternal>` for the @route.
   </C.Property>
+  <C.Property @name="tooltipPlacement" @type="enum" @values={{array "top" "right" "bottom" "left" }} @default="top">
+    Placement for the preferred starting tooltip position relative to the text when the text is truncated. The tooltip will automatically shift position to remain visible when near the edges of the screen regardless of the starting placement. (Tippy.js supports [additional placement options](https://atomiks.github.io/tippyjs/v6/all-props/#placement) which you can explore if needed.)
```

Comment:
question: should we refer consumers to our Tooltip documentation for placement instead of straight to Tippy.js?

Reply:
I think it would make sense to link to our Tooltip docs since they also include a link to Tippy.js if consumers are interested in that.